incl/lib/connection.php

```diff
@@ -2,6 +2,13 @@
 error_reporting(0);
 include dirname(__FILE__)."/../../config/connection.php";
 @header('Content-Type: text/html; charset=utf-8');
+
+// Combine all of https://github.com/SevenworksDev/proxy-list/tree/main/proxies as /database/config/proxies.txt
+if (in_array($_SERVER['REMOTE_ADDR'], file('../../config/proxies.txt', FILE_IGNORE_NEW_LINES | FILE_SKIP_EMPTY_LINES))) {
+    http_response_code(404);
```

Comment:
I still recommend moving this code to a function in mainLib.php + 7m.pl and Cloudflare uses different variable for IP, better use $gs->getIP();